app/models/invitation.rb

```diff
@@ -290,29 +290,33 @@ def self.invite_initial_employee!(census_employee)
   end
 
   def self.invite_broker!(broker_role)
-    return nil unless should_invite_broker_or_broker_staff_role?(broker_role)
-
-    invitation = self.create(
-      :role => "broker_role",
-      :source_kind => "broker_role",
-      :source_id => broker_role.id,
-      :invitation_email => broker_role.email_address
-    )
-    invitation.send_broker_invitation!(broker_role.parent.full_name)
-    invitation
+    if should_invite_broker_or_broker_staff_role?(broker_role)
+      invitation = self.create(
+        :role => "broker_role",
+        :source_kind => "broker_role",
+        :source_id => broker_role.id,
+        :invitation_email => broker_role.email_address
+      )
+      invitation.send_broker_invitation!(broker_role.parent.full_name)
+      invitation
+    elsif should_notify_linked_broker_or_broker_staff_role?(broker_role)
```

Comment:
@TreyE I think this should be as simple as if-else. If they are eligible for `Notification`, we will send them an email notification; if not, we will send an `Invitation`. Thoughts?